client/blocks/login/index.jsx

```diff
@@ -138,12 +138,16 @@ class Login extends Component {
 	}
 
 	render() {
-		const { translate, twoStepNonce } = this.props;
+		const { translate, twoFactorEnabled, twoFactorAuthType } = this.props;
 
 		return (
 			<div>
 				<div className="login__form-header">
-					{ twoStepNonce ? translate( 'Two-Step Authentication' ) : translate( 'Log in to your account.' ) }
+					{
+						twoFactorEnabled && twoFactorAuthType
+							? translate( 'Two-Step Authentication.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Two Step Authentication Enabled' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/14039)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).